dev/CommonStyles/CommonStyles.vcxitems

```diff
@@ -154,13 +154,23 @@
     <Page Include="$(MSBuildThisFileDirectory)TextBox_themeresources.xaml">
       <UseVisualStyle>Latest</UseVisualStyle>
       <Version>RS1</Version>
-      <Type>ThemeResources</Type>
+      <Type>ThemeResources</Type> 
     </Page>
     <Page Include="$(MSBuildThisFileDirectory)TextBox_themeresources_v2.5.xaml">
       <UseVisualStyle>V2dot5</UseVisualStyle>
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
+    <Page Include="$(MSBuildThisFileDirectory)TextBlock_themeresources.xaml">
+      <UseVisualStyle>Latest</UseVisualStyle>
```

Comment:
You just need one file. For example, TextBlock_any_themeresources.xaml
Then 
```
<UseVisualStyle>Any</UseVisualStyle>
```

Reply:
I don't think that we can if we want to preserve old styling on v2.5 as @ranjeshj suggested [here](https://github.com/microsoft/microsoft-ui-xaml/pull/4108#discussion_r570985718)